src/plan.rs

```diff
@@ -0,0 +1,1161 @@
+// Miniscript
+// Written in 2022 by rust-miniscript developers
+//
+// To the extent possible under law, the author(s) have dedicated all
+// copyright and related and neighboring rights to this software to
+// the public domain worldwide. This software is distributed without
+// any warranty.
+//
+// You should have received a copy of the CC0 Public Domain Dedication
+// along with this software.
+// If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
```

Comment:
We have SPDX identifiers now, this can all be removed and just plain old:
```
// SPDX-License-Identifier: CC0-1.0
```

Reply:
As of 4849893 this is still TODO.

Reply:
Solved in 9a88807fda085e76b638cd66f4c35b7559b97a8a in #592